src/App/Repository/Config/features.php

```diff
@@ -85,4 +85,9 @@
 	'hxl' => [
 		'enabled' => false,
 	],
+
+	// Enable or disable User Settings feature
+	'user-settings' => [
+		'enabled' => true,
```

Comment:
This should be disabled by default. We need to add a way to override per environment but for now we should always be keeping things disabled unless we want them on in production.